ioctl/client_test.go

```diff
@@ -129,7 +130,7 @@ func TestGetAddress(t *testing.T) {
 
 func TestNewKeyStore(t *testing.T) {
 	r := require.New(t)
-	testWallet, err := os.MkdirTemp(os.TempDir(), "ksTest")
+	testWallet, err := os.MkdirTemp(os.TempDir(), "testKeyStore")
```

Comment:
please go over all the test.go files to make sure no 2 tests are using the same temp file name

Reply:
yes, no same temp dir name.